README.md

```diff
@@ -1,73 +1,27 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# Hello Microverse Project
 
+> This project tests my understanding on tools and best practices i have learned so far. For instance, gitHub flow, create branched with different base branches, concept of version control, open pull request in gitHub, write descriptive commit messages and ability to submit a project for code review.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
```

Comment:
- [ ] You README file is descriptive, you did well, but please add another session for getting started to show how use your project.